src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,136 @@
+package duke;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+
+/**
+ * Handles the read and write operations to the data file
+ */
+public class Storage {
+
+    private String path;
+    private String content = "";
+
+    /**
+     * Constructor for the Storage Class
+     *
+     * @param path relative filepath where the datafile is located
+     */
+    public Storage(String path) {
+        this.path = path;
+    }
+
+    /**
+     * Reads the datafile if it exists else it creates a new data file
+     * Adds all the tasks on the datafile into a new ArrayList
+     *
+     * @return returns a list of Task
+     * @throws DukeException exception to be thrown if there is error reading the file
+     */
+    public List<Task> load() throws DukeException {
```

Comment:
I think the method can rename with more specific description such as loadsData for clarity. I noticed the same issues in several other places too.